downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md

```diff
@@ -50,8 +50,34 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Update 9 (2201.9.0)](
 
 ### New features
 
-#### `avro` package
+#### `persist` package
+- Incorporated support for the PostgreSQL data store, mirroring the functionality provided for other supported SQL data stores like MySQL and MSSQL.
+- Implemented support for the `Redis` data store, including the following features:
+  - Support optional fields to be defined in the data model, providing flexibility in structuring data.
```

Comment:
Since this is a list of items after we say " including the following features:", shouldn't these be
```suggestion
  - Support for optional fields to be defined in the data model, providing flexibility in structuring data.
```

Reply:
Fixed in [70ca3c7](https://github.com/ballerina-platform/ballerina-dev-website/pull/8881/commits/70ca3c7b388d2526aa364810f78adc5a3f47dd67)